README.md

```diff
@@ -32,6 +86,152 @@ vantage_password: [my_password]
+tend to have larger houses.
+
+  **WARNING**: At the top of the Light panel is a very tempting, innocuous
+  looking switch.  It turns on (or off) *all* the lights in your home at the
+  same time.  Turning on all your lights simultaneously IS A BAD IDEA.
+
+  Why?  If you have LED lighting, the internal capacitors in those lights take a
+  bunch of current to charge when you first turn them on.  If you turn on enough
+  of them all on at once, the load spike may be large enough to pop several
+  breakers or worse.
+
+One of the first things you will be strongly tempted to do is manually configure
+your Lovelace UI to clean out all the stuff you don't care about.  This is not a
+bad idea.
+
+So what has the Vantage component added to your home assistant?  A number of
+things:
+
+1.  For every light controlled by Vantage, a light.* entity.  Lights can be a
+    dimmable or non-dimmable, color (DMX) or monochrome.
+2.  For every switched load controlled by Vantage, a switch.* entity.
+3.  For every window shade controlled by Vantage, a cover.* entity.
+4.  For every motion sensor, contact sensor (such as a door opening sensor) or
+    other binary sensor attached to Vantage, a sensor.* entity.
+5.  Temperature, power and current sensors showing the status of the Infusion
+    dimmer modules and their attached loads.
+6.  For every light group defined by Vantage, a light.* entity.
+7.  For every button on every keypad, a sensor.* entity showing whether the
+    button is current pressed or not.
+8.  For every keypad, a sensor.* entity saying which button on the keypad was
+    pressed last.
+9.  For every variable defined in Vantage, a sensor.* entity containing the
+    current state of that variable.
+10. Services for setting Vantage variables and calling Vantage tasks
+    (vantage.set_variable, vantage.call_task, vantage.set_variable_vid, and
+    vantage.call_task_vid).
+11. Events which get fired whenever a keypad button is pressed
+    (vantage_button_pressed, vantage_button_released,
+    vantage_button_multipressed).
+
+
+Naming of Entities
+==================
+
+Every entity in Home Assistant needs a unique name.  In Vantage, objects don't
+need unique names -- they all have a unique VID.
+
+This Vantage component attempts to generate unique names by concatenating the
+names of the levels of the object hierarchy in Vantage.  So a light in your
+second floor master bath might get called
+"light.second_floor_master_bath_vanity_light".  If you have two objects with the
+same name (very common with buttons, where multiple buttons may have similar
+names such as "On" and "Off"), this module will just add the VID to the end.
+
+These names are very long.  If you want shorter names, you can either go through
+and rename all the Entities in Home Assistant, or you can rename your objects in
+Vantage before syncing to Home Assistant (say, by renaming your "Second Floor"
+to be "2F" and your "Master Bath" to be "MBath" or the like).
```

Comment:
Oooh, I'd not noticed that feature.

Strangely the 'area' field is case sensitive -- in that if you put any capital letters in it at all it will never match anything.  I think we can fix that bug.  ;-)

Also, this appears to only update the Friendly name of Entities, not the annoying long symbol names you get for them if your names in Vantage are overly crazy verbose like mine are.  :-)

Updated description here.